packages/aws-cdk/test/integ/helpers/cdk.ts

```diff
@@ -239,6 +304,85 @@ export class TestFixture {
       ...this.fullStackName(stackNames)], options);
   }
 
+  public async cdkBootstrapLegacy(options: CdkLegacyBootstrapCommandOptions = {}): Promise<string> {
+    const args = ['bootstrap'];
+
+    if (options.verbose) {
+      args.push('-v');
+    }
+    if (options.toolkitStackName) {
+      args.push('--toolkit-stack-name', options.toolkitStackName);
+    }
+    if (options.bootstrapBucketName) {
+      args.push('--bootstrap-bucket-name', options.bootstrapBucketName);
+    }
+    if (options.noExecute) {
+      args.push('--no-execute');
+    }
+    if (options.publicAccessBlockConfiguration !== undefined) {
+      args.push('--public-access-block-configuration', options.publicAccessBlockConfiguration.toString());
+    }
+    if (options.tags) {
+      args.push('--tags', options.tags);
+    }
+    if (options.region) {
+      args.push(`aws://${await this.aws.account()}/${options.region}`);
+    }
+
+    return this.cdk(args, {
+      ...options.cliOptions,
+      modEnv: {
+        ...options.cliOptions?.modEnv,
+        // so that this works for V2,
+        // where the "new" bootstrap is the default
+        CDK_LEGACY_BOOTSTRAP: '1',
+      },
+    });
+  }
+
+  public async cdkBootstrap(options: CdkBootstrapCommandOptions = {}): Promise<string> {
```

Comment:
Since this is forcing "modern" bootstrap, maybe put that in the name as well.

Are we not testing *default* bootstrap anywhere?

Reply:
Done.